export-helpers/README.md

```diff
@@ -0,0 +1,28 @@
+# export-helpers
+
+## What it does
+
+This extension demonstrates how to use export helpers in Firefox to share
+JavaScript objects defined in content scripts with scripts loaded by web pages.
+
+It has a background script that does two things:
+
+* execute a content script in the current tab, when the user clicks a browser action
+* listen for messages from the content script, and display a notification when the message arrives.
+ 
+The content script then:
+ 
+ * defines a function `notify()` and exports that to the page as a property
```

Comment:
it could makes sense to explain that the first one  `exportFunction` and the second one `cloneInto with cloneFunctions option`.

Reply:
If possible, I'd like to get this PR finished off, and we could look at this afterwards?